src/Command/CleanMediaCommand.php

```diff
@@ -26,13 +29,37 @@
 /**
  * @final since sonata-project/media-bundle 3.21.0
  */
-class CleanMediaCommand extends ContainerAwareCommand
+class CleanMediaCommand extends Command
 {
     /**
      * @var MediaProviderInterface[]|null
      */
     private $providers;
 
+    /**
+     * @var Pool
+     */
+    private $mediaPool;
+
+    /**
+     * @var Local
+     */
+    private $filesystemLocal;
+
+    /**
+     * @var MediaManagerInterface
+     */
+    private $mediaManager;
+
+    public function __construct(Local $filesystemLocal, Pool $mediaPool, MediaManagerInterface $mediaManager)
```

Comment:
... and this, but I think it's okay for a command if we communicate this in the Changelog and Upgrade file